examples/test.md

```diff
@@ -1,7 +1,7 @@
 ---
 runme:
   id: 01HF7B0KJM3HHFDVSXA6E12JM3
-  version: v2.0
+  version: v2.2
```

Comment:
@pastuxso Is this intended?

Reply:
Yes, it doesn't matter, The idea is to track the versión :wink: